sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -18,7 +18,9 @@
           with EventData.
   -->
   <ItemGroup>
-    <ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" />
+    <!--ProjectReference Include="..\..\..\core\Azure.Core.Experimental\src\Azure.Core.Experimental.csproj" /-->
```

Comment:
why not just delete?

Reply:
Because we'll need to flip it back after release due to another round of changes.